packages/office-ui-fabric-react/src/components/DetailsList/DetailsRowCheck.types.ts

```diff
@@ -71,3 +76,7 @@ export interface IDetailsRowCheckStyles {
   check: IStyle;
   isDisabled: IStyle;
 }
+
+export interface IDetailsCheckboxProps {
```

Comment:
Rather than redefine `checked`, can we replace this interface with `Pick<ICheckProps, 'checked'>`? This way there is a relationship enforced.